src/constants/formData.js

```diff
@@ -0,0 +1,7 @@
+import { HIGH, MEDIUM, LOW } from "./priority";
+
+export const selectPriorityData = [
```

Comment:
[NITPICK] selectPriorityData => PRIORITY_OPTIONS. Constants should be named in MACRO_CASE